content/70.sustainability.md

```diff
@@ -1 +1,85 @@
+Expanding the featureset of yt tends to be more attractive than optimization for a large swath of the userbase; in some sense, optimization opens new avenues for scientific exploration (by taking formerly out-of-reach options and placing them within the realm of reasonable execution time), but adding new features certainly does.
+
+A few examples of new functionality provided by community members include the volume rendering of octree datasets, support for new code frontends (such as AREPO), and many of the analysis modules.
+New functionality provided by funded development has included long-term improvements such as the "demeshening" of particle datasets and support for non-astrophysical domains.
+A few notable improvements to the efficiency and scalability of yt include multi-level parallelism operations and the initial implementation of dask as an array backend.
+
+In general, we have found it difficult to move through large upheavals of code without large-scale effort on the part of the community.
+This has resulted in a reluctance to investigate particularly invasive changes, and can provide a distortion of the cost/benefit analysis, leaning toward risk aversion.
+The YTEP process mitigates this somewhat, but also provides additional opportunities for a 'veto' that can dampen enthusiasm and impede development.
+At the same time, the safe guards provided by this can help to ensure stability for the community, particularly those not actively engaged in development.
+
+### Project Maintenance
+
+In addition to development of new features, the correction of problems or updates to old behavior are also critical to the sustained usage and applicability of a project.
+In this category of "bug fixes" we certainly include faults or problems with yt itself, but it's also important to note that in many cases, we can also use this category to describe improvements in behavior that bring the results greater accuracy or precision.
+
+It is tough to estimate the cost of bug fixes, in terms of labor or in terms of community member attrition, but @doi:10.1007/s11219-021-09564-z suggests that large, established projects on github may spend up to 40% of their time on bugfixes rather than new features.
+While we have not been able to verify this, examining the cadence of bug fix releases versus new feature releases suggests that this is indeed the case for yt as well.
+
+Maintenance and bug fixes are critical to developing yt as a community-focused code.
+The damage that can be done by ignoring a bug report, dismissing a suggestion, or outright rejecting proposed fixes is real, and can have measurable impact on community cohesion and growth.
+As such, in the yt development process we strive to be as accommodating and welcoming as possible, while still balancing that against the very real costs that responding to bug fixes, feature requests, and providing helpful insight with issues that may be covered by the documentation or that are known bugs.
```

Comment:
I *think* this sentence isn't finished ? I can't seem to get the hang of it

Reply:
Can you take another look?